src/lib/rmt/ssl/certificate_generator.rb

```diff
@@ -59,6 +59,9 @@ def ca_present?
   end
 
   def server_cert_present?
+    # NB this doesn't check the second file if the first one exists
+    # An improvement would be to look for the absence of any ssl configuration and proceed in that case,
+    # but leave ssl alone if any configuration (hand-edited or incomplete) is
```

Comment:
The last sentence incomplete seems, as if by Yoda master written it was. :grimacing: